plugin/secrets_access_token.go

```diff
@@ -91,6 +93,17 @@ func secretAccessTokenRevoke(ctx context.Context, req *logical.Request, d *frame
 		return logical.ErrorResponse(fmt.Sprintf("revoke returned error: %v", err)), nil
 	}
 	if err := googleapi.CheckResponse(resp); err != nil {
+		gErr, ok := err.(*googleapi.Error)
+		if !ok {
+			return logical.ErrorResponse(err.Error()), nil
+		}
+		if gErr.Code == 400 && strings.Contains(gErr.Body, "invalid_token") {
+			// The token is no longer valid, probably because it was revoked internally
+			// Mark as revoked but return a warning.
+			return &logical.Response{
```

Comment:
It's going to make the code more complicated, but I think we should add another field to the schema that's like `ignore_failed_revocation` that is false by default. 

If that field is set, print the warning.

If that field is not set, return an error with a suggestion to set that field on the request.

Reply:
sgtm

Reply:
On second thought, I think this is something that should be expected - it's likely Vault will always attempt to revoke the lease and will get this error as long as the user doesn't revoke it during its lifetime. Updated to just check that this doesn't happen and ignore it.